ML-Frameworks/pytorch-aarch64/get-source.sh

```diff
@@ -84,11 +80,27 @@ git-shallow-clone https://github.com/pytorch/pytorch.git $PYTORCH_HASH
     # Not used for CPU only builds
     git rm third_party/cudnn_frontend
     git rm third_party/cutlass
+    git rm third_party/flash-attention
     git rm third_party/NVTX
+    # Only used in WIN32 builds (see CMakeLists.txt:394)
+    git rm third_party/mimalloc
 
     # Update submodules
     git submodule sync
     git submodule update --init --checkout --force --recursive --jobs=$(nproc)
+
+    # Remove deps that we don't need which come from third party. It would be nice to avoid
+    # fetching completely, but this was tricky with git submodule update --init --checkout --force --recursive 
+    (
+        cd third_party/fbgemm
+        git rm external/cutlass
+        git rm external/composable_kernel
+    )
+    (
+        cd third_party/aiter
+        git rm 3rdparty/composable_kernel
+    )
+
```

Comment:
Minor request: can we add a 

```bash
python3 build_bundled.py 
```

after pulling sources to update the `LICENSES_BUNDLED.txt`?